src/lib/pmd/PmdEngine.ts

```diff
@@ -77,11 +76,7 @@ export class PmdEngine implements RuleEngine {
 		try {
 			const targetPaths: string[] = [];
 			for (const target of targets) {
-				if (target.isDirectory) {
-					targetPaths.push(...target.paths.map(p => path.join(target.target, p)));
-				} else {
-					targetPaths.push(...target.paths);
-				}
+				targetPaths.push(...target.paths);
```

Comment:
These are always absolute paths. No need to handle paths separately. 

Now that I think of it, I wonder if we can remove isDirectory from the RuleTarget, I will look into this.